.github/actions/test-server/action.yaml

```diff
@@ -41,15 +41,21 @@ runs:
         username: ${{ github.actor }}
         password: ${{ inputs.ghcr-pat }}
 
+    # We can't use a make target here because a composite action
+    # doesn't have a .git folder when checked out.
     - name: Start server based on released version
       if: ${{ inputs.jimm-version != 'dev' }}
-      run: make integration-test-env
+      run: |
+        cd local/traefik/certs; ./certs.sh; cd - && \
+        docker compose --profile test up -d --wait
       shell: bash
+      working-directory: ${{ github.action_path }}/../../..
```

Comment:
Second answer mentions that it should be possible to set a default work dir for each step in a action:
https://stackoverflow.com/questions/58139175/running-actions-in-another-directory

Reply:
Unfortunately it's not supported in composite actions, see Github's [design doc here](https://github.com/actions/runner/blob/main/docs/adrs/0549-composite-run-steps.md#defaults).

Reply:
in a follow up PR we will try the default on top of the job list, like so:
```
name: CI

on:
 ...

defaults:
 run:
  working-directory: ./app
```